cmd/zc_enumerator.go

```diff
@@ -659,9 +659,24 @@ func passedFilters(filters []objectFilter, storedObject storedObject) bool {
 	return true
 }
 
+// This error should be treated as a flag, that we didn't fail processing, but instead, we just didn't process it.
+// Currently, this is only really used for symlink processing, but it _is_ an error, so it must be handled in all new traversers.
+// Basically, anywhere processIfPassedFilters is called, additionally call getProcessingError.
+var ignoredError = errors.New("FileIgnored")
+
+func getProcessingError(errin error) (ignored bool, err error) {
+	if errin == ignoredError {
+		return true, nil
+	}
+
+	return false, err
+}
+
 func processIfPassedFilters(filters []objectFilter, storedObject storedObject, processor objectProcessor) (err error) {
 	if passedFilters(filters, storedObject) {
 		err = processor(storedObject)
+	} else {
+		err = ignoredError
```

Comment:
I'm a bit confused here, the comment above says that the ignoredError is only used for symlink processing, but here it appears that any storedObject that didn't pass the filters (e.g. include/exclude) would get it. 

I'd imagine that we'd

Reply:
We've clarified the implementation offline: the ignoredError is all about the situation where the include/exclude flag interact with a folder and a symlink that points to it; they need to be considered separately.